python/ray/experimental/channel/torch_tensor_type.py

```diff
@@ -58,6 +59,10 @@ def __init__(
                 shape; if it does not match, the task will error.
             dtype: The expected dtype of the torch.Tensor. Similar to the
                 shape, this may be statically or dynamically declared.
+            direct_return: Whether the tensor is sent directly or inside of
```

Comment:
QQ: do we have a test that raises an exception when direct_return = True but it actually doesn't do it?

Reply:
Hmm we have one for DAG inputs but I think not for outputs sent between actors. Let me add.